### PR TITLE
Update first-time-setup.md

### PR DESCRIPTION
**Pull request by neeta-wagento**
## This PR is a: Fix typos
- [ ] New topic
- [ ] Content fix or rewrite
- [x] Bug fix or improvement
 

## Summary
 Fix typos
 
When this pull request is merged, it will...

### CODE REVIEW

**Comment by shrielenee** [comment]
You should add changes from PR #2267 to this PR, and push back up for review. Please also ensure you're changing all applicable versions (if they aren't symlinked together).

File: guides/v2.1/cloud/access-acct/first-time-setup.md [hunk 1]
```diff
@@ -40,10 +40,10 @@ Prior to setting up your workspace, you may want to gather the following credent
...
 * **Cloud Project account**: The Project Owner or Technical Admin (Super User) should invite you to the {{site.data.var.ece}} project. When you receive the e-mail invitation, click the link and follow the prompts to create your account. See [Set up an account]({{ page.baseurl }}/cloud/before/before-workspace.html#newaccount) for details.
-* **Magento Encryption Key**: When importing an existing Magento system only, you will need to capture the Magento encryption key used to protect your access and data for the Magento database. For details on this key, see [Resolve issues with encryption key]({{ page.baseurl }}/cloud/trouble/trouble-crypt-key-variable.html)
+* **Magento Encryption Key**: When importing an existing Magento system only, you will need to capture the Magento encryption key used to protect your access and data for the Magento database. For details on this key, see [Resolve issues with an encryption key]({{ page.baseurl }}/cloud/trouble/trouble-crypt-key-variable.html)
```
Comment (shrielenee) [inline guides/v2.1/cloud/access-acct/first-time-setup.md]: This change isn't relevant, as "Resolve issues with encryption key" is the actual name of the page. Please revert.

**Comment by neeta-wagento** [comment]
Hello @shrielenee 
I have added the changes! Kindly review and confirm. 

Thank you!

**Comment by shrielenee** [comment]
Thanks for making these changes @neeta-wagento!

**Pull request merged**